README.md

```diff
@@ -1,6 +1,6 @@
 # Katello
 
-[![Build Status](https://ci.theforeman.org/buildStatus/icon?job=test_katello)](https://ci.theforeman.org/job/test_katello)
+[![Build Status](https://ci.theforeman.org/buildStatus/icon?job=katello-nightly-release)](https://ci.theforeman.org/job/katello-nightly-release/)
```

Comment:
Is this the right jenkins job?

Reply:
This would technically be the release job, but i think that is okay!